docs/pages/utilities.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Utility libraries
+layout: default
+---
+
+# Utility libraries
+
+Useful packages for 'utility' functionality, normally we highlight these if
+we've found a library that we prefer to Python's builtin. You probably don't
+want to write these yourself but might need logging or a command-line interface.
+
+## Command-line interface
+
+| Name                                                        | Short description                                                              | 🚦  |
+| ----------------------------------------------------------- | ------------------------------------------------------------------------------ | :-: |
+| [typer](https://typer.tiangolo.com/)                        | A user-friendly and intuitive CLI system. Built on click.                      | 🟢  |
+| [click](https://click.palletsprojects.com/)                 | Makes nice-looking CLIs, and more configurable than argparse. Uses decorators. | 🟠  |
+| [argparse](https://docs.python.org/3/library/argparse.html) | Python's builtin CLI system uses object configuration.                         | 🟠  |
+| [optparse](https://docs.python.org/3/library/optparse.html) | A now-deprecated CLI system built into Python.                                 | 🔴  |
```

Comment:
If it's deprecated, why have it?

Reply:
Because it still exists (it's possible to `import optparse` ) we want people to _know_ not to use it.

>  We actively discourage using this. This could be because it’s no longer maintained, not open source, or difficult to use. Consider moving to alternatives if you’re currently using something that’s red. A reason for not using this is given.